crontab

```diff
@@ -1,2 +1,3 @@
 # reload SSL certificate around every four hours, in case it was renewed by acme.sh
 0 */4 * * * /home/monitoring/reload-cert.sh > /home/monitoring/last_cert_reload.log 2>&1
+0 */4 * * * /home/monitoring/backup.sh production
```

Comment:
Should we rather run this in a pipeline where we can see it's red when the backup failed?

Reply:
I think you can:  https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

Reply:
Then I'd be fine with trying that out. As suggested we should pick a weird time though to preventthe job from being dropped under high load.